__tests__/integration.test.js

```diff
@@ -118,3 +119,16 @@ describe("/api/articles", () => {
       });
   });
 });
+
+describe.skip('/api/articles/:article_id/comments', () => {
```

Comment:
Assuming this is skipped because it's functionality for the ticket you started whilst waiting on feedback (so I won't look at it)